db/schema.rb

```diff
@@ -10,11 +10,55 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_07_10_005441) do
+ActiveRecord::Schema.define(version: 2021_07_26_001500) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "volunteers", id: :serial, force: :cascade do |t|
+    t.string "name"
+    t.string "email"
+    t.string "slackUserId"
+    t.string "pronouns"
+    t.string "primarySkills"
+    t.string "secondarySkills"
+    t.string "employer"
+    t.string "student"
+    t.string "assignedProjects"
```

Comment:
This seems redundant to me with the volunteer_projects table, but maybe I'm misinterpreting the purpose.

Reply:
Yeah, along with the skills columns on this table. Will remove.